src/ddoc_cache/test/eunit/ddoc_cache_no_cache_test.erl

```diff
@@ -35,45 +35,43 @@ return_error(_DDocId) ->
     {error, timeout}.
 
 
-start(Resp) ->
-    Ctx = ddoc_cache_tutil:start_couch(),
+no_cache_test_() ->
```

Comment:
Same comment as before, please leave these test changes in a separate commit from the others for posterity (don't squash all together)

Reply:
Good change tho